doc/release-notes-15437.md

```diff
@@ -0,0 +1,54 @@
+P2P and network changes
+-----------------------
+
+
+The command line option to enable BIP61 (`-enablebip61` has been removed).
+Thus, Bitcoin Core no longer sends reject messages and applications that
+require this feature should use the recommended alternatives:
+
+[ See below ]
+
+Mailing list post [DO NOT INCLUDE IN RELEASE NOTES]
+-----------------
+
+Subject: Removal of reject network messages from Bitcoin Core (BIP61)
+
+Bitcoin Core may send "reject" messages as response to "tx", "block" or
+"version" messages from a network peer when the message could not be accepted.
+
+This feature is toggled by the `-enablebip61` command line option and has been
+disabled by default since Bitcoin Core version 0.18.0 (not yet released as of
+time of writing). Nodes on the network can not generally be trusted to send
+valid ("reject") messages, so this should only ever be used when connected to a
+trusted node. At this time, I am not aware of any software that requires this
+feature, and I would like to remove if from Bitcoin Core to make the codebase
+slimmer, easier to understand and maintain. Let us know if your application
+relies on this feature and you can not use any of the recommended alternatives:
+
+* Testing or debugging of implementations of the Bitcoin P2P network protocol
+  should be done by inspecting the reject messages that are produced by a
```

Comment:
Should this say "inspecting the logs ..."?

Might be useful to have an example log here.

Reply:
I guess an example log depends on the message type that was rejected "version", "tx" or "block" and the reason for rejection. I'd rather not add a specific example, since the logs are not guaranteed to be stable (only used for testing and debugging)

Reply:
I think it'd still be helpful, even with a caveat that log messages are not guaranteed to be stable. Obviously up to you!